launch/client.py

```diff
@@ -449,6 +451,7 @@ def create_model_endpoint(
         gpu_type: Optional[str] = None,
         endpoint_type: str = "sync",
         update_if_exists: bool = False,
+        child_fn_info: Optional[Dict[str, Any]] = None,
```

Comment:
not the graph necessary but the step:
`{"step_name": {"type": "sync", "destination": "endpoint_name" } }`

Reply:
Can't the bundle (pipeline) contain both the manifest of steps, as well as their relationships (the graph)? Or rather, can't it just store the graph, since you can derive the manifest from the graph (but not vice versa)?

Reply:
I guess deriving the manifest on either the client or server technically works though..